rpc/cloudagent/network/networkinterface/moc_cloudagent_networkinterface.proto

```diff
@@ -40,7 +41,7 @@ message IpConfiguration {
 	repeated InboundNatRule inboundNatRules = 8;
 	Tags tags = 9;
 	NetworkType networkType = 10;
-	NetworkSecurityGroupReference networksecuritygroup = 11;
+	NetworkSecurityGroupReference networkSecurityGroupRef = 12;
```

Comment:
Can you briefly explain why do we need to do this?

Reply:
We tried just renaming, but it seems like something in deserialization will still pick up that they're the "same thing". The issue that we're resolving is I mistakenly thought we hadn't released with NSG without lnet but we had. because of that, if someone had saved data from a March build and tried to upgrade, their data wouldn't deserialize, for NSG, VNIC or VNET. By changing the name, and the message number (and keeping the existing message number reserved), we can make sure anyone who upgrades will keep their data.

Reply:
got it! thanks!